packages/driver/cypress/e2e/commands/assertions.cy.js

```diff
@@ -820,23 +820,8 @@ describe('src/cy/commands/assertions', () => {
         done()
       })
 
-      cy.get('body').then(() => {
-        expect(cy.currentSubject()).to.match('body')
-      })
-    })
-
-    it('sets type to child when subject matches', (done) => {
```

Comment:
`log:added` is in our public API in case anyone reading this was wondering: https://docs.cypress.io/api/events/catalog-of-events#Cypress-Events But I agree this is extremely unlikely to cause breakage.